core/capabilities/remote/executable/request/server_request.go

```diff
@@ -32,14 +32,22 @@ type srMetrics struct {
 }
 
 func (s *srMetrics) recordExecutionDuration(ctx context.Context, d time.Duration, success bool) {
+	successStr := "false"
+	if success {
+		successStr = "true"
+	}
 	s.executeDuration.Record(ctx, d.Milliseconds(), metric.WithAttributes(
-		attribute.Bool("success", success), attribute.String("callingDON", s.callingDonID), attribute.String("capabilityID", s.capabilityID),
+		attribute.String("success", successStr), attribute.String("callingDON", s.callingDonID), attribute.String("capabilityID", s.capabilityID),
```

Comment:
why is it necessary to use a String?

Reply:
Curious about this too

Reply:
Non string attributes don't show up properly and show up as NOT_SET

Reply:
This is a bug that the Beholder team are aware of